modules/hub/src/ChannelsService.ts

```diff
@@ -335,10 +335,11 @@ export default class ChannelsService {
   ): Promise<WithdrawalArgs | null> {
     const channel = await this.channelsDao.getChannelByUser(user)
     if (!channel || channel.status !== 'CS_OPEN') {
-      throw new Error(
+      LOG.error(
```

Comment:
In a couple cases, normal operation will result in the hub throwing an error & printing out a bug ugly stack track. I replaced this with a `LOG.error` followed by a `return`. I assume this'll work the same but be a little quieter.